### ISSUE TITLE
Undocumented handling of routing-controller errors

### ISSUE THREAD

**Issue by PhilippMolitor** [epiphone/routing-controllers-openapi#66]
I implemented this library and immediately noticed that there is no example or documentation on how to include routing-controller's `HttpError`s in the generated spec.
Is there a defined best-practice how to add the spec for a method that is able to throw e.g. a `NotFoundError`?

**Comment by epiphone** [comment]
No, there isn't. You could use the `OpenAPI` decorator to add a `responses.404` property, or perhaps write another decorator that calls `OpenAPI` internally to save a bit of typing.

Please share here if you come up with a better solution!

**Comment by PhilippMolitor** [comment]
Is there any plan on supporting that? This seems like a core component to me. Imagine `@ErrorResponseSchema(NotFoundError)`.

**Comment by epiphone** [comment]
No plans, nope. PRs are welcome :+1:

**Comment by bratanon** [comment]
+1 for this feature
